guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,232 @@
+
+```diff
+- channel.send("content", { reply: '123456789012345678' })` // User ID
++ channel.send("content", { replyTo: '765432109876543219' })` // Message ID
+```
+
+The new `MessageOptions.allowedMentions.repliedUser` boolean option determines if the reply will notify the author of the original message.
+
+```diff
+- message.reply('content')
++ message.reply('content', { allowedMentions: { repliedUser: false }})
+```
+
+### Bitfields / Permissions
+
+Bitfields such as Permissions and UserFlags will no longer support the use of string literals, instead requiring use of the flag.
+
+```diff
+- permissions.has('SEND_MESSAGES')
++ permissions.has(Permissions.FLAGS.SEND_MESSAGES)
+```
+
+### Webpack
+
+Webpack builds are no longer supported.
+
+## Breaking changes and deletions
+
+### Guild
+
+#### Guild#member
+
+The helper/shortcut method `Guild#member()` has been removed.
+
+```diff
+- guild.member(user);
++ guild.members.cache.get(user.id)
+```
+
+#### Guild#voice
+
+The `Guild#voice` getter has been removed.
+
+```diff
+- guild.voice
++ guild.me.voice
+```
+
+### GuildMember / GuildMemberManager
+
+#### GuildMember#ban / GuildMemberManager#ban
+
+`GuildMember#ban()` and `GuildMemberManager#ban` will throw a TypeError when a string is provided instead of an options object.
+
+```diff
+- member.ban('reason')
++ member.ban({ reason: 'reason' })
+
+- guild.members.ban('123456789012345678', 'reason')
++ guild.members.ban('123456789012345678', { reason: 'reason' })
+```
+
+### Message / MessageManager
+
+#### Message#delete / MessageManager#delete
+
+`Message.delete()` and `MessageManager.delete()` no longer accept any options, requiring a timed-delete to be performed manually.
+
+```diff
+- message.delete({ timeout: 10000 });
++ client.setTimeout(() => message.delete(), 10000);
+```
+
+### RoleManager
+
+#### RoleManager#fetch
+
+`RoleManager.fetch()` will now return a Collection instead of a RoleManager when called without params.
+
+### User
+
+#### User#locale
+
+`User.locale` has been removed, as this property is not exposed to bots.
+
+### UserFlags
+
+The deprecated UserFlags `DISCORD_PARTNER` and `VERIFIED_DEVELOPER/EARLY_VERIFIED_DEVELOPER` have been removed in favour of their renamed versions.
+
+```diff
+- user.flags.has(UserFlags.FLAGS.DISCORD_PARTNER)
++ user.flags.has(UserFlags.FLAGS.PARTNERED_SERVER_OWNER)
+
+- user.flags.has(UserFlags.FLAGS.VERIFIED_DEVELOPER)
++ user.flags.has(UserFlags.FLAGS.EARLY_VERIFIED_BOT_DEVELOPER)
+```
+
+## Additions
+
+### ActivityTypes
+
+New activity type `COMPETING` added.
+
+### Channel
+
+#### Channel#isText()
+
+The new `Channel#isText()` getter provides an easy way for TypeScript developers to determine if a channel is Text-Based ("dm", "text", "news")
+
+### ClientOptions
+
+#### ClientOptions#messageEditHistoryMaxSize
+
+The new `ClientOptions#messageEditHistoryMaxSize` option allows developers to set a fixed limit on the edit history stored for each message in the client cache. 
```

Comment:
This was a v12 option which is removed in v13 -- v13 no longer stores message edit history at all.